versions.yaml

```diff
@@ -0,0 +1,25 @@
+module-sets:
+  edot-base:
+    version: v0.124.0
+    modules:
+      - github.com/elastic/opentelemetry-collector-components/receiver/elasticapmreceiver
+      - github.com/elastic/opentelemetry-collector-components/receiver/loadgenreceiver
+      - github.com/elastic/opentelemetry-collector-components/processor/elasticinframetricsprocessor
+      - github.com/elastic/opentelemetry-collector-components/processor/elastictraceprocessor
+      - github.com/elastic/opentelemetry-collector-components/processor/lsmintervalprocessor
+      - github.com/elastic/opentelemetry-collector-components/processor/ratelimitprocessor
+      - github.com/elastic/opentelemetry-collector-components/connector/elasticapmconnector
+
+excluded-modules:
+  - github.com/elastic/opentelemetry-collector-components/internal/tools
+  - github.com/elastic/opentelemetry-collector-components/internal/sharedcomponent
+  - github.com/elastic/opentelemetry-collector-components/internal/testutil
+  - github.com/elastic/opentelemetry-collector-components/pkg/integrations
+  - github.com/elastic/opentelemetry-collector-components/receiver/integrationreceiver
+  - github.com/elastic/opentelemetry-collector-components/extension/fileintegrationextension
+  - github.com/elastic/opentelemetry-collector-components/extension/configintegrationextension
+  - github.com/elastic/opentelemetry-collector-components/extension/beatsauthextension
+  - github.com/elastic/opentelemetry-collector-components/extension/apmconfigextension
+  - github.com/elastic/opentelemetry-collector-components/processor/integrationprocessor
```

Comment:
Curious, what is the reasoning behind excluding these modules?

Reply:
I am not aware of their usage outside this repository, nor have been released in the past https://github.com/elastic/opentelemetry-collector-components/tags

I would add them to the list as needed, would you prefer to release them all?

Reply:
I was not aware that they are not used outside. Sounds good to me to release as needed.